code/Test_Definitions/NumberVerification_verify.feature

```diff
@@ -0,0 +1,137 @@
+    And NUMBERVERIFY_VERIFY_MATCH_PHONENUMBER1 is compliant with the schema DevicePhoneNumber
+    And NUMBERVERIFY_VERIFY_MATCH_PHONENUMBER2 is compliant with the schema DevicePhoneNumber
+    And NUMBERVERIFY_VERIFY_MATCH_PHONENUMBER1 is different to NUMBERVERIFY_VERIFY_MATCH_PHONENUMBER2
+
+  @NumberVerification_verify0_phoneNumber_does_not_match_schema
+  Scenario Outline: phoneNumber value does not comply with the schema
+    Given the request body property "$.phoneNumber" is set to: <phone_number_value>
+    When the HTTP "POST" request is sent
+    Then the response status code is 400
+    And the response property "$.status" is 400
+    And the response property "$.code" is "INVALID_ARGUMENT"
+    And the response property "$.message" contains a user friendly text
+
+    Examples:
+      | phone_number_value |
+      | string_value       |
+      | 1234567890         |
+      | +12334foo22222     |
+      | +00012230304913849 |
+      | 123                |
+      | ++49565456787      |
+
+  @NumberVerification_verify1_xcorrelator_does_not_match_schema
+  Scenario Outline: x-correlator request header value does not comply with the schema
+    Given if the optional request header "x-correlator" is set to: <xcorrelator_value>
+    When the HTTP "POST" request is sent
+    Then the response status code is 400
+    And the response property "$.status" is 400
+    And the response property "$.code" is "INVALID_ARGUMENT"
+    And the response property "$.message" contains a user friendly text
+
+    Examples:
+      | xcorrelator_value  |
+      | string_value       |
+      | boink              |
+
+
+  @NumberVerification_verify100_match_true
+  Scenario:  verify phone number NUMBERVERIFY_VERIFY_MATCH_PHONENUMBER1, network connection and access token matches NUMBERVERIFY_VERIFY_MATCH_PHONENUMBER1
+    Given they use the base url over a mobile connection
+    And the resource is "/verify"
+    And they acquired a valid access token associated with NUMBERVERIFY_VERIFY_MATCH_PHONENUMBER1 through OIDC authorization code flow or CIBA
+    And one of the scopes associated with the access token is number-verification:verify
+    When the HTTPS "POST" request is sent
+    And the mobile connection is associated with NUMBERVERIFY_VERIFY_MATCH_PHONENUMBER1
+    And the request body has the field phoneNumber with a value of NUMBERVERIFY_VERIFY_MATCH_PHONENUMBER1
+    And the response header "x-correlator" has same value as the request header "x-correlator"
+    And the response header "Content-Type" is "application/json"
+    And the response body complies with the OAS schema at "/components/schemas/SendCodeResponse"
+    Then the response status code is 200
+    And the response property "$.devicePhoneNumberVerified" is true
+
+
+  @NumberVerification_verify101_match_false
+  Scenario:  verify phone number NUMBERVERIFY_VERIFY_MATCH_PHONENUMBER1, network connection and access token matches NUMBERVERIFY_VERIFY_MATCH_PHONENUMBER2
```

Comment:
Should this scenario title be a "does not match" instead of a "match".

Suggested change to scenario title: `verify phone number NUMBERVERIFY_VERIFY_MATCH_PHONENUMBER1, network connection and access token does not match NUMBERVERIFY_VERIFY_MATCH_PHONENUMBER2`

Reply:
Changed it. But different to your suggestion. Better?

Reply:
Yes, looks good to me